### PR TITLE
Add support for new revision of Govee H5054 water leak detector

### LINKED ISSUE #2265: Unable to read Govee Leak Detector Sensors
I purchased two packs of the Govee H5054 leak detector sensors, one [3-pack with the Wifi gateway included](https://www.amazon.com/dp/B07J9HZ5VN), and one ["expansion pack" with just five of the bare sensors](https://www.amazon.com/dp/B07QSFRSJX). 

I already have a working instance of rtl_433 connected to Home Assistant for pulling data from some Acurite sensors, and was able to easily add the three detectors that came with the gateway, but have not been able to read anything from the five that came in the expansion pack. All five seemingly function perfectly, and I was even able to add them to the gateway, but no luck with RTL_433.

Opening them up, both versions have a date code of `2021-08-10`, and board revision `V5.2`, so unless Govee changed the firmware I'm at a loss. I'm happy to provide any additional info or logs that might help.

### PR DESCRIPTION
**Pull request by jasonmichalski**
This adds support for the updated protocol used by newer revisions of the Govee H5054 water leak detector.

Closes #2265

### CODE REVIEW

File: src/devices/govee.c [hunk 1]
```diff
@@ -248,3 +251,101 @@ r_device govee = {
...
         .fields      = output_fields,
 };
+
+typedef enum {
+    GOVEE_BUTTON_PRESS   = 0,
+    GOVEE_BATTERY_REPORT = 1,
+    GOVEE_WATER_LEAK     = 2,
+} govee_h5054_event_t;
+
+static int govee_decode_h5054(r_device *decoder, bitbuffer_t *bitbuffer)
```
Comment (zuckschwerdt) [inline src/devices/govee.c:261]: name it govee_h5054_decode, add a doc-comment

File: include/rtl_433_devices.h [hunk 2]
```diff
@@ -231,6 +231,7 @@
     DECL(fineoffset_wn34) \
     DECL(rubicson_pool_48942) \
     DECL(badger_orion) \
+    DECL(govee_h5054)
```
Comment (zuckschwerdt) [inline include/rtl_433_devices.h:234]: needs a `\` at the end

Comment (jasonmichalski) [reply include/rtl_433_devices.h]: fyi - it seems like clang-format strips it out, but i added it back in manually

**Comment by zuckschwerdt** [comment]
Yes, this is good. To be merged soon.

**Comment by drew-hill** [comment]
I'm unable to pull in any Protocols above 223 on v22.11 on my computer or on my separate Home Assistant device. 

Instead, I receive the error:  "Protocol number specified (231) is larger than number of protocols" 

Has anyone else had this problem while trying to read from the Govee 5054 Protocol 231? I'm considering starting an issue -- search currently pulls up 0 matches in Issues at the moment -- but imagine I'm probably just doing something stupid as an rtl_433 newbie.

**Comment by drew-hill** [comment]
This is exactly what I meant re: newbie growing pains. Thanks!

**Pull request merged**